portfolio/src/main/webapp/script.js

```diff
@@ -57,3 +57,37 @@ function getText() {
 }
 
 
+//wk3pt6
+
+
+function getComments() {
```

Comment:
This should be renamed.  I would expect "getComments" to return the comments, not insert them into the HTML.  Something like "populateComments" would be more appropriate.

Also, add a comment describing the function and any preconditions

Reply:
Resolving because function name was updated